pkg/reconciler/ingress/lister.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2019 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"go.uber.org/zap"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/sets"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"knative.dev/serving/pkg/apis/networking/v1alpha1"
+	istiolisters "knative.dev/serving/pkg/client/istio/listers/networking/v1alpha3"
+	"knative.dev/serving/pkg/network"
+	"knative.dev/serving/pkg/network/ingress"
+	"knative.dev/serving/pkg/network/status"
+)
+
+// NewStatusProber creates a new instance of status.Prober
+func NewStatusProber(
```

Comment:
the name of the function does not really feel right for the file name

Reply:
Done

Reply:
It's still `NewStatusProber` in the `lister.go` file